snmplib/tools.c

```diff
@@ -1413,3 +1413,12 @@ netsnmp_string_time_to_secs(const char *time_string) {
     return secs;
 }
 #endif /* NETSNMP_FEATURE_REMOVE_STRING_TIME_TO_SECS */
+
+char *netsnmp_gethomedir() {
```

Comment:
Why the return type `char *` instead of `const char *`? Callers of this function are not expected to modify the string returned by this function, isn't it?

Reply:
Done